authors/apps/articles/serializers.py

```diff
@@ -86,3 +89,20 @@ def get_slug(self, obj):
 
     def get_description(self,obj):
        return obj.article.description
+
+
+class ArticleRatingSerializer(serializers.ModelSerializer):
+
+    title = serializers.SerializerMethodField()
+    author = serializers.SerializerMethodField()
+    rating = serializers.IntegerField(required=True, max_value=5, min_value=1)
```

Comment:
@calvinpete please set the min_value to zero

Reply:
Okay.